### DISCUSSION TITLE
Deprecation Warning: Passing percentage units to the global abs() function is deprecated.

### DISCUSSION THREAD

**Discussion by suigewoaini00** [twbs/bootstrap#39034] [Q&A] [answered]
Deprecation Warning: Passing percentage units to the global abs() function is deprecated.
In the future, this will emit a CSS abs() function to be resolved by the browser.
To preserve current behavior: math.abs(100%)
To emit a CSS abs() now: abs(#{100%})

I can't create a bootstarp project. This is the warning that comes up when I introduce css, the css style can't be loaded.

**Comment by ZuBB** [comment, marked as answer]
https://github.com/twbs/bootstrap/issues/39028

(_probably this should be closed_)
